internal/archiver/archiver.go

```diff
@@ -78,10 +78,19 @@ type Archiver struct {
 	// WithAtime configures if the access time for files and directories should
 	// be saved. Enabling it may result in much metadata, so it's off by
 	// default.
-	WithAtime   bool
-	IgnoreInode bool
+	WithAtime bool
+
```

Comment:
I added this because the previous one has a multi-line comment, which suggests that it also relates to the next option. They're quite unrelated, though: WithAtime is about what information to store, while ChangeIgnore is about how to detect changes.